### PR TITLE
Nsstress fix

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Fix nstress Test 
1- Move Test in  yaml file
2- remove unused code
3- rectify as move process.run to process.system
rename yaml file as align with unique across repository

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: generic/nstress.py.data/nstress.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@ tar_ball_ubuntu: "nstress_Ubuntu1410_ppc64_Nov_2015.tar"
...
-memchunk: "250"
-memchunk_nmem64: "2047"
-procs: "255"
+test: !mux
+        nmem:
+                testcase: "nmem"
+                memchunk: "250"
+        nmem64:
+                testcase: "nmem64"
+                memchunk: "2047"
```
Comment (narasimhan-v) [inline generic/nstress.py.data/nstress.yaml]: @PraveenPenguin procs ?

File: generic/nstress.py [hunk 2]
```diff
@@ -54,23 +44,22 @@ def setUp(self):
...
+
         ''' ncpu retrun code is 1 even after successful completion'''
-        ncpu_result = process.run("./ncpu -p %s -s %s" %
-                                  (self.procs, self.duration),
-                                  ignore_status=True, sudo=True)
-        if ncpu_result.exit_status != 1:
+
+        if process.system("/ncpu -p %s -s %s" %
+                          (self.procs, self.duration),
+                          ignore_status=True, sudo=True) != 1:
```
Comment (narasimhan-v) [inline generic/nstress.py]: Are we not failing here ?

Comment (PraveenPenguin) [reply generic/nstress.py]: checked with @Pavithra1602 and it should be fail , will push the changes

**Comment by PraveenPenguin** [comment]
@narasimhan-v pushed the changes , please have a look

File: generic/nstress.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def test(self):
         if process.system("/ncpu -p %s -s %s" %
                           (self.procs, self.duration),
                           ignore_status=True, sudo=True) != 1:
-            self.log.info("ncpu test failed")
+            self.fail("ncpu test failed")
```
Comment (narasimhan-v) [inline generic/nstress.py]: @PraveenPenguin this should be in the other commit.
Also, these all changes can be in a single commit.

Comment (PraveenPenguin) [reply generic/nstress.py]: @narasimhan-v pushed the changes

File: generic/nstress.py.data/nstress.yaml [hunk 4]
```diff
@@ -10,3 +10,4 @@ test: !mux
         nmem64:
                 testcase: "nmem64"
                 memchunk: "2047"
+
```
Comment (narasimhan-v) [inline generic/nstress.py.data/nstress.yaml]: @PraveenPenguin white space needs to be removed please

Comment (PraveenPenguin) [reply generic/nstress.py.data/nstress.yaml]: pushed the changes please have a look

File: generic/nstress.py [hunk 5]
```diff
@@ -54,24 +44,23 @@ def setUp(self):
...
+            self.fail("%s test failed " % self.test)
+
         ''' ncpu retrun code is 1 even after successful completion'''
-        ncpu_result = process.run("./ncpu -p %s -s %s" %
-                                  (self.procs, self.duration),
-                                  ignore_status=True, sudo=True)
-        if ncpu_result.exit_status != 1:
-            self.log.info("ncpu test failed")
+
+        if process.system("/ncpu -p %s -s %s" %
```
Comment (narasimhan-v) [inline generic/nstress.py]: @PraveenPenguin just noticed, it should be `./ncpu`

**Comment by narasimhan-v** [comment]
@PraveenPenguin please add job.log

File: generic/nstress.py (hunk 5 above)
Comment (PraveenPenguin) [reply generic/nstress.py]: ah thanks  i miss that while doing rebase : (

**Comment by PraveenPenguin** [comment]
[root@system ~]# avocado run nstress.py -m nstress.py.data/nstress.yaml 
JOB ID     : fea748062d8a277b3ed7b1f629fe4ee8c127c024
JOB LOG    : /var/lib/libvirt/images/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-06-19T00.27-fea7480/job.log
 (1/2) nstress.py:NStress.test;nmem-71ba: PASS (1024.25 s)
 (2/2) nstress.py:NStress.test;nmem64-145d: PASS (1168.18 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 2193.12 s
JOB HTML   : /var/lib/libvirt/images/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-06-19T00.27-fea7480/results.html
[root@system ~]# 

2019-06-19 00:45:01,881 parameters       L0146 DEBUG| PARAMS (key=duration, path=*, default=300) => '500'
2019-06-19 00:45:01,881 parameters       L0146 DEBUG| PARAMS (key=testcase, path=*, default=nmem) => 'nmem64'
2019-06-19 00:45:01,882 parameters       L0146 DEBUG| PARAMS (key=memchunk, path=*, default=250) => '2047'
2019-06-19 00:45:01,882 parameters       L0146 DEBUG| PARAMS (key=procs, path=*, default=250) => '255'
2019-06-19 00:45:01,882 process          L0626 INFO | Running './nmem64 -m 2047 -s 500'
2019-06-19 00:56:09,762 process          L0458 DEBUG| [stdout] size 2096128.00K mixed - ops= 1737162K, time=  502.52 ops/s=    3457K
2019-06-19 00:56:09,762 process          L0714 INFO | Command './nmem64 -m 2047 -s 500' finished with 0 after 667.877114058s
2019-06-19 00:56:09,796 process          L0626 INFO | Running './ncpu -p 255 -s 500'
[root@system ~]# tail -f /var/lib/libvirt/images/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-06-19T00.27-fea7480/job.log
2019-06-19 00:45:01,831 test             L0298 DEBUG| DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
2019-06-19 00:45:01,833 parameters       L0146 DEBUG| PARAMS (key=tar_ball_rhel, path=*, default=nstress_RHEL71_LE_ppc64_Nov_2015.tar) => 'nstress_RHEL71_LE_ppc64_Nov_2015.tar'
2019-06-19 00:45:01,881 parameters       L0146 DEBUG| PARAMS (key=duration, path=*, default=300) => '500'
2019-06-19 00:45:01,881 parameters       L0146 DEBUG| PARAMS (key=testcase, path=*, default=nmem) => 'nmem64'
2019-06-19 00:45:01,882 parameters       L0146 DEBUG| PARAMS (key=memchunk, path=*, default=250) => '2047'
2019-06-19 00:45:01,882 parameters       L0146 DEBUG| PARAMS (key=procs, path=*, default=250) => '255'
2019-06-19 00:45:01,882 process          L0626 INFO | Running './nmem64 -m 2047 -s 500'
2019-06-19 00:56:09,762 process          L0458 DEBUG| [stdout] size 2096128.00K mixed - ops= 1737162K, time=  502.52 ops/s=    3457K
2019-06-19 00:56:09,762 process          L0714 INFO | Command './nmem64 -m 2047 -s 500' finished with 0 after 667.877114058s
2019-06-19 00:56:09,796 process          L0626 INFO | Running './ncpu -p 255 -s 500'
^C
[root@system~]#

**Pull request merged**